### PR TITLE
Added additional_jumpboost support, added and fixed getvel support

### PR DESCRIPTION
**Pull request by TheProJamer**
- Fixed HL2 2707 getvel pattern sometimes not being found.
- Added additional_jumpboost (formerly additional_abh) support to:
    - Half-Life Source (build 2257546)
    - Portal 2 (build 6879)
    - Half-Life 2 (builds 2707, 4044, 5135 and 2257546), Half-Life 2 Episode One (builds 4044, 5135 and 2257546), Half-Life 2 Episode Two (builds 4104 and 2257546) and Portal (builds 5135 and 2257546) in the rare scenario that someone wishes to play multiplayer with the jumpboost mechanics present.
- Added getvel support to:
    - Half-Life 2 (builds 2707 and 4044)
    - Half-Life 2 Episode One (build 4044)
    - Portal 2 (build 6879)

### CODE REVIEW

File: spt/OrangeBox/modules/ServerDLL.cpp [hunk 1]
```diff
@@ -440,7 +473,7 @@ bool __fastcall ServerDLL::HOOKED_CheckJumpButton_Func(void* thisptr, int edx)
 
 void __fastcall ServerDLL::HOOKED_FinishGravity_Func(void* thisptr, int edx)
 {
-	if (insideCheckJumpButton && y_spt_additional_abh.GetBool())
+	if (insideCheckJumpButton && y_spt_additional_jumpboost.GetInt())
```
Comment (YaLTeR) [inline spt/OrangeBox/modules/ServerDLL.cpp]: Why did you change this to `GetInt()`? It's identical to `GetBool()` here, and `GetBool()` is better.

File: spt/OrangeBox/modules/ServerDLL.cpp [hunk 2]
```diff
@@ -451,7 +484,7 @@ void __fastcall ServerDLL::HOOKED_FinishGravity_Func(void* thisptr, int edx)
 			AngleVectors(mv->m_vecViewAngles, &vecForward);
 			vecForward.z = 0;
 			VectorNormalize(vecForward);
-
+
```
Comment (YaLTeR) [inline spt/OrangeBox/modules/ServerDLL.cpp]: Extra whitespace.

File: spt/OrangeBox/modules/ServerDLL.cpp [hunk 3]
```diff
@@ -206,15 +206,40 @@ void ServerDLL::Hook(const std::wstring& moduleName, HMODULE hModule, uintptr_t
 			break;
 
 		case 1:
+			off1M_bDucked = 2;
```
Comment (YaLTeR) [inline spt/OrangeBox/modules/ServerDLL.cpp]: Why did the `case 1:` offsets change?

Comment (TheProJamer) [reply spt/OrangeBox/modules/ServerDLL.cpp]: Maybe because you never added a pattern to go with those offsets in the first place, and so it was pointless to keep.
